mainapp/templates/mainapp/request_form.html

```diff
@@ -3,7 +3,47 @@
 {% load static %}
 
 {% block content %}
-  <form method="post"  class="simple-form">
+
+<script type="text/javascript">
+
+function countDigits(string) {
+    var count = (string.match(/\d/g) || []).length
+    return count
+}
+
+function validateForm() {
+    var isFormValid = true;
+    var phoneNumberErrorMessage = "";
+    var requestee_phone = $('#id_requestee_phone').val();
+
+    if(countDigits(requestee_phone) > 10) {
```

Comment:
There are already PR based on #457 to make phone max length 12. You might want to consider that.
Also, the if/else flow can be made more performance optimized by using nested checks and using or/and conditions.